inc/class-statify-dashboard.php

```diff
@@ -301,8 +301,9 @@ public static function get_stats( $force_refresh = false ) {
 	/**
 	 * Get stats from DB
 	 *
-	 * @since    0.1.0
-	 * @version  1.4.0
+	 * @since   0.1.0
+	 * @since   1.4.0
+	 * @since   2.0.0  Add search queries to `$data` array.
```

Comment:
I guess we used the `@version` wrong, if I am right, it is ment to set versions for APIs and similar (https://docs.phpdoc.org/guide/references/phpdoc/tags/version.html#version), and you can use multiple `@since` (https://docs.phpdoc.org/guide/references/phpdoc/tags/since.html#since)

Reply:
Yep, `@since` is commonly used for changelog purposes, mostly changes to method signature or high-level behavior (at least helpful on public API that can be quite helpful to have this information on hand in the IDE)
`@version` is meant for semantic changes and the state of the functionality.

Multiple of either one don’t make much sense, because they yield barely any information, just “something was done here“.

Something like this would be more or less meaningful:
```
@since 1.0
@since 1.1 return boolean result instead of void
@since 1.2 make parameter $foo optional with default 42

@version 1.1
```

Reply:
"Since" we only use it on some functions and we don't even have a real "public API", would we even mind keeping these `@since` version numbers up to date?

Reply:
Well, a public method is a public method and there are some recommendations out there for things like workarounds or custom triggers that call some of these. Some snippets given by ourselves in the support forums. Documenting API changes enables users of those to potentially migrate their logic without opening another support request, if changes are not self-explaining.

I agree, the majority of our code is designed for internal use and only exposed public for hooks etc.

But as with every project, that’s yet another part of documentation that might get out of sync over time. Though it’s probably one of the easier parts, as old annotations typically never become wrong, so we only have to append another one-liner, if necessary.